cookbooks/aws-parallelcluster-install/recipes/intel.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: intel
+#
+# Copyright 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+if node['cluster']['base_os'] == 'centos7'
+  bash 'download dependencies Intel HPC platform' do
+    code <<-INTEL
+      yum install --downloadonly compat-libstdc++-33 nscd nss-pam-ldapd openssl098e --downloaddir=#{node['cluster']['intelhpc']['dependencies_dir']}
+      yum-config-manager --add-repo https://yum.repos.intel.com/hpc-platform/el7/setup/intel-hpc-platform.repo
```

Comment:
where is this coming from?

Reply:
No where. Sorry that I was confused by various tests I did and thought it was necessary